src/components/PrettyTime.jsx

```diff
@@ -5,6 +5,7 @@ import PropTypes from 'prop-types';
 const PrettyTime = (props) => {
   const { isoTime } = props;
 
+  moment.locale('en-US');
```

Comment:
Force US locale for this component so we don't get weird behavior like random Polish dates on a Polish computer.